src/main/java/io/datanerds/avropatch/schema/CustomTypes.java

```diff
@@ -0,0 +1,87 @@
+package io.datanerds.avropatch.schema;
+
+import io.datanerds.avropatch.Patch;
+import io.datanerds.avropatch.value.conversion.BigDecimalConversion;
+import io.datanerds.avropatch.value.conversion.BigIntegerConversion;
+import io.datanerds.avropatch.value.conversion.DateConversion;
+import io.datanerds.avropatch.value.conversion.UUIDConversion;
+import org.apache.avro.LogicalType;
+import org.apache.avro.LogicalTypes;
+import org.apache.avro.Schema;
+import org.apache.avro.Schema.Type;
+import org.apache.avro.SchemaBuilder;
+
+import java.util.Arrays;
+
+import static io.datanerds.avropatch.schema.OperationTypes.NAMESPACE;
+import static io.datanerds.avropatch.schema.PrimitiveTypes.*;
+import static org.apache.avro.Schema.createRecord;
+import static org.apache.avro.Schema.createUnion;
+
+/**
+ * This interface holds the schema constants for all supported custom types.
+ *
+ * @see UUIDConversion
+ * @see DateConversion
+ * @see BigDecimalConversion
+ * @see BigIntegerConversion
+ * @see io.datanerds.avropatch.operation.Path
+ */
+public interface CustomTypes {
+
+    interface PatchType {
+        String NAME = Patch.class.getSimpleName();
+        String DOC = "This record represents a PATCH holding a sequence of operations to apply to a given object.";
+        static Schema create(Schema valueSchema) {
+            Schema.Field operations = new Schema.Field("operations",
```

Comment:
Better to move the literal ("operations") to a constant before using it here and other places.